Dockerfile

```diff
@@ -1,11 +1,5 @@
 FROM ghcr.io/foundry-rs/foundry:master as builder
 
-# Install the Rust toolchain.
```

Comment:
Hmm, this is actually going to have some consequences for how we do testnet deployments. It’s fine to remove, but we will need a new solution. I’ll work with @mcclurejt to get this figured out before the next release, but I just wanted to make a note of it.